source/fs.c

```diff
@@ -56,6 +88,17 @@ size_t FileWrite(void* buf, size_t size, size_t foffset)
     return bytes_written;
 }
 
+bool DebugFileWrite(void* buf, size_t size, size_t foffset) {
+    size_t bytesWritten = FileWrite(buf, size, foffset);
+    if(bytesWritten != size) {
+        Debug("ERROR, SD card may be full!");
+        FileClose(); // <- need to keep this in mind!
```

Comment:
It was convenient, but you're right. It is bad coding style. Fixed in the last commit.